locales/en/messages.json

```diff
@@ -1813,6 +1813,9 @@
     "dialogCopyProfileClose": {
         "message": "Cancel"
     },
+    "dialogCalibrateCancel": {
```

Comment:
This makes me wonder how many different cancel translations we have. I'll take a look and do a PR for it. No worry for now.

Reply:
I can imagine! I'm actually not using this anymore, I've now removed it. It was left over from when I once opened the calibrator in a modal.

Reply:
@Docteh: I think a 'deduplicate translations where reasonable' pull request is in order. I was going to do this, so let me know if you are already working on this.

Reply:
@mikeller I haven't started on it, have at it.